test/Synths.test.ts

```diff
@@ -0,0 +1,187 @@
+import { ethers, waffle } from "hardhat";
```

Comment:
what is the difference `Synths` and `SynthsSdk` files? i can think of Synths as the Sdk and the opposite, the naming on these should be more accurate

Reply:
```Synths.test``` tests a lspfpl synth. What do you think about ``Àsset.test``` instead?
Or should we name it ```LSPFPL.test```?

Reply:
good to separate them each into its own file
- `EMP-Synths.test`
- `FPL-Synths.test`
- `LSP-Synths.test`